lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -0,0 +1,56 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      SET_INFO_PLIST_VALUE_CUSTOM_VALUE = :SET_INFO_PLIST_VALUE_CUSTOM_VALUE
```

Comment:
You don't set a value for `SET_INFO_PLIST_VALUE_CUSTOM_VALUE` anywhere. That's okay, you can just remove `SET_INFO_PLIST_VALUE_CUSTOM_VALUE` from the `SharedValues` module.